terraform/dns_nodejs_org.tf

```diff
@@ -0,0 +1,348 @@
+  name    = "unofficial-builds"
+  proxied = false
+  ttl     = 1
+  type    = "A"
+  value   = "45.55.98.129"
+  zone_id = "1206c4f949d69993ae55d9d015804406"
+}
+
+resource "cloudflare_record" "terraform_managed_resource_39362d1838ae02ce3c212d8cf1d122a5" {
+  name    = "nodejs.org"
+  proxied = true
+  ttl     = 1
+  type    = "AAAA"
+  value   = "2604:a880:400:d1::a3c:f001"
+  zone_id = "1206c4f949d69993ae55d9d015804406"
+}
+
+resource "cloudflare_record" "terraform_managed_resource_b92bb39ac8d8a7fe1f60f295ae4e6f3c" {
+  name    = "_2ad3ed9cacf2d25d5224d04db1a4d677"
+  proxied = false
+  ttl     = 1
+  type    = "CNAME"
+  value   = "2fb3dc900f0968bd883ee25a88024c39.45e0122a0374b8b09f485b2d239c01b7.e16c1c41bcd4c7842b.sectigo.com"
+  zone_id = "1206c4f949d69993ae55d9d015804406"
+}
+
+resource "cloudflare_record" "terraform_managed_resource_475548ad57e9d2b01393a7e543231148" {
+  name    = "_c44b831a22ab9039953536a5fbf41513"
+  proxied = false
+  ttl     = 1
+  type    = "CNAME"
+  value   = "e7e4215c01a2775a2d2fd7b799eeb963.8d7d7eb200163a2107dc4d4505aa08ba.fa3ed62adf396f9275.sectigo.com"
+  zone_id = "1206c4f949d69993ae55d9d015804406"
+}
+
+resource "cloudflare_record" "terraform_managed_resource_6ab233c3c1b231e6bb576df49cd80e8e" {
+  name    = "_c9ef34ab7f60f3e351aaeaa158a3553c"
+  proxied = false
+  ttl     = 1
+  type    = "CNAME"
+  value   = "c2b1a295cdb3f1c5cfa03d1bc7c6b598.0ef9b3ba802051f61c70bc3080ec968f.w0834777001502709292.comodoca.com"
+  zone_id = "1206c4f949d69993ae55d9d015804406"
+}
+
+resource "cloudflare_record" "terraform_managed_resource_b5355dea1fae185149c7c970ef36c74e" {
+  name    = "coverage"
+  proxied = true
+  ttl     = 1
+  type    = "CNAME"
+  value   = "nodejs.org"
+  zone_id = "1206c4f949d69993ae55d9d015804406"
+}
+
+resource "cloudflare_record" "terraform_managed_resource_e23716010539dc90edf5667f022f1fee" {
+  name    = "foundation"
+  proxied = true
+  ttl     = 1
+  type    = "CNAME"
+  value   = "nodejs.org"
+  zone_id = "1206c4f949d69993ae55d9d015804406"
+}
+
+resource "cloudflare_record" "terraform_managed_resource_fafc41f07e8294ec9bc6dc4b7e46c976" {
+  name    = "interactive"
+  proxied = true
+  ttl     = 1
+  type    = "CNAME"
+  value   = "nodejs.org"
+  zone_id = "1206c4f949d69993ae55d9d015804406"
+}
+
+resource "cloudflare_record" "terraform_managed_resource_7ee0bf83cdbf7c81f1474ad30e474e3c" {
+  name    = "live"
+  proxied = true
+  ttl     = 1
+  type    = "CNAME"
+  value   = "nodejs.github.io"
+  zone_id = "1206c4f949d69993ae55d9d015804406"
+}
+
+resource "cloudflare_record" "terraform_managed_resource_e7385cf24ad8660f929ad64132c56c72" {
+  name    = "logs"
+  proxied = true
+  ttl     = 1
+  type    = "CNAME"
+  value   = "logs.libuv.org"
+  zone_id = "1206c4f949d69993ae55d9d015804406"
+}
+
+resource "cloudflare_record" "terraform_managed_resource_d0dbc6d7694d2445b23e84f5fdc8e6cd" {
+  name    = "modules"
+  proxied = false
+  ttl     = 1
+  type    = "CNAME"
+  value   = "npmjs.org"
```

Comment:
Aside: that seems like something that should transfer to GitHub/NPM @MylesBorins